media/js/firefox/australis/tour.js

```diff
@@ -3,6 +3,12 @@
 
     var firstTime = 'True';
 
+    // GA tracking for Firstrun tests
```

Comment:
Do we also need to ensure that f=34 here, or are we ok with everyone getting these events?